### PR TITLE
Uberftp 2.8

### LINKED ISSUE #41277: uberftp fails to build on 10.10.3
I installed *all* formulae from the core, this is one of the issues I ran into. I’m at work, will check tonight.

```
$ brew install uberftp
==> Installing uberftp dependency: globus-toolkit
…
==> Installing uberftp
==> Downloading https://github.com/JasonAlt/UberFTP/archive/Version_2_7.tar.gz
Already downloaded: /Library/Caches/Homebrew/uberftp-2.7.tar.gz
sh: /usr/local/opt/globus-toolkit/sbin/gpt-query: No such file or directory
==> ./configure --prefix=/usr/local/Cellar/uberftp/2.7 --with-globus-flavor= --with-globus=/usr/loca
checking for awk... awk
checking whether make sets $(MAKE)... yes
configure: creating ./config.status
config.status: creating pkg_data_src.gpt
configure: error: --with-gpt=PATH must specify a valid path no

READ THIS: https://git.io/brew-troubleshooting
```

Logs: https://gist.github.com/dc88a8103e33c01c6d81

### PR DESCRIPTION
**Pull request by maarten-k**
Update uberftp to version 2.8 based on work done by giffels (closed pull request #40848).
* Closes  #41277
* The patch provided at JasonAlt/UberFTP#8 makes the situation worse: another patch is needed to get ./configure running  and cause more obfuscation than the 2 replace commands. 
* Replaced the single quotes in formula for double quotes

### CODE REVIEW

File: Library/Formula/uberftp.rb [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
   def install
-    # get the flavor
     globus = Formula["globus-toolkit"].opt_prefix
 
-    core = `"#{globus}/sbin/gpt-query" globus_core`
-    flavor = case core
-    when /gcc64dbg/ then "gcc64dbg"
-    when /gcc32dbg/ then "gcc32dbg"
-    end
+    #patches needed since location changed with globus-toolkit versions >= 6.0, patch to upstream not yet merged https://github.com/JasonAlt/UberFTP/pull/8, but solves not whole problem
```
Comment (mikemcquaid) [inline Library/Formula/uberftp.rb]: Add space after `#` and wrap this line at 80 characters.

File: Library/Formula/uberftp.rb [hunk 2]
```diff
@@ -27,3 +23,4 @@ def install
     system "#{bin}/uberftp", "-v"
   end
 end
+
```
Comment (mikemcquaid) [inline Library/Formula/uberftp.rb]: Don't need this empty line

**Comment by apjanke** [comment]
Looks good now. Could you squash this to a single commit so we have a tidy commit history?

**Comment by maarten-k** [comment]
I will try to squash the commits today or tomorrow. Please bear with me: my experience with git is limited.

**Comment by mikemcquaid** [comment]
@maarten-k I'll squash them for you.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by maarten-k** [comment]
@mikemcquaid Thanks for squashing and for maintaining this awesome project!

**Comment by maarten-k** [comment]
Closing, since the pull request is committed in master 03a12ec24bd9f9299e50d5eae00a115c84a71fe6

**Pull request closed without merging**